spec/stack_master/sparkle_formation/compile_time/allowed_values_validator_spec.rb

```diff
@@ -1,30 +1,31 @@
 RSpec.describe StackMaster::SparkleFormation::CompileTime::AllowedValuesValidator do
+
```

Comment:
I thought it was out, but you pulled it back in

Reply:
just making them all the same